editor/test/state.js

```diff
@@ -9,7 +9,7 @@ import deepFreeze from 'deep-freeze';
  * Internal dependencies
```

Comment:
Do we want unit tests for `editor.edits`, `saving`, and `currentPost` reducer behaviors?